test/e2e/apimachinery/resource_quota.go

```diff
@@ -1419,7 +1419,7 @@ var _ = SIGDescribe("ResourceQuota [Feature:PodPriority]", func() {
 
 })
 
-var _ = SIGDescribe("ResourceQuota [Feature:CrossNamespacePodAffinity] [alpha]", func() {
+var _ = SIGDescribe("ResourceQuota [Feature:CrossNamespacePodAffinity]", func() {
```

Comment:
to run the test, the Feature tag needs to be dropped.

Reply:
also could you remove the feature tag from the [alpha test job](https://github.com/kubernetes/test-infra/blob/4ba52b738449601577ef3d91ad8db4a38c42adf6/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L426)?

Reply:
> to run the test, the Feature tag needs to be dropped.

Are you sure? I don't see the tag is dropped for other features (see `Feature:ScopeSelectors` in the same file)

Reply:
> also could you remove the feature tag from the alpha test job?

Yes, but that needs to be done in a followup PR since it is a different repo.

Reply:
> Are you sure? I don't see the tag is dropped for other features (see Feature:ScopeSelectors in the same file)

That means those are not getting tested in default e2e runs either. `[Feature:...]` tags are excluded from default e2e runs. When the features are enabled by default, we drop the tag.

Reply:
sg, removed the feature tag and squashed the commits.

also set https://github.com/kubernetes/test-infra/pull/22184 to remove the test from the alpha job